frontend/public/components/about-modal.tsx

```diff
@@ -26,6 +36,51 @@ import {
   hasAvailableUpdates,
 } from '../module/k8s/cluster-settings';
 
+const DynamicPlugins: React.FC = () => {
+  const { t } = useTranslation();
+  const [pluginInfoEntries, allPluginsProcessed] = useDynamicPluginInfo();
+  const [rows, setRows] = React.useState([]);
+
+  React.useEffect(() => {
+    const loadedPlugins = pluginInfoEntries.filter(isLoadedDynamicPluginInfo);
+    setRows(
+      loadedPlugins?.map((plugin) => {
+        return {
+          cells: [plugin.metadata.name, plugin.metadata.version],
+        };
+      }),
+    );
+  }, [pluginInfoEntries]);
+
+  const headers = [
+    {
+      title: t('public~Name'),
+    },
+    {
+      title: t('public~Version'),
+    },
+  ];
+
+  return allPluginsProcessed ? (
```

Comment:
if one plugin is Failed/Pending, we will always show loading icon, right? 
Can we just show the plugin has been successfully loaded here?

Reply:
https://user-images.githubusercontent.com/12692381/203736774-9198c5b8-9534-4181-a38e-391c56e30b50.mov

Reply:
@vojtechszocs, see above.  Since `Pending` is a legit plugin status, https://github.com/openshift/console/blob/master/frontend/packages/console-plugin-sdk/src/api/useDynamicPluginInfo.ts#L37 is incorrect, no?

Reply:
It occurs to me I really don't need to use `allPluginsProcessed` and will remove it to avoid the bug @yapei noted.

Reply:
> It occurs to me I really don't need to use `allPluginsProcessed` and will remove it to avoid the bug @yapei noted.

Changes pushed.